pkg/controllers/provisioningv2/rke2/rkecontrolplane/controlplane.go

```diff
@@ -32,25 +36,39 @@ func Register(ctx context.Context, clients *wrangler.Context) {
 
 	rkecontrollers.RegisterRKEControlPlaneStatusHandler(ctx, clients.RKE.RKEControlPlane(),
 		"", "rke-control-plane", h.OnChange)
-	relatedresource.Watch(ctx, "rke-control-plane-trigger", func(namespace, name string, obj runtime.Object) ([]relatedresource.Key, error) {
```

Comment:
This didn't work because the management cluster and the control plane don't have the same name and namespace.